### ISSUE TITLE
Link to the SCOP information for Fold data labels 

### ISSUE THREAD

**Issue by ghost** [multicom-toolbox/DeepSF#1]
Hi :)

I am working with the [remote homology dataset from TAPE](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7774645/)  which uses the SCOP 1.75 dataset described in the [DeepSF paper (sections 2.1.1, suppl. I.1.1)](https://academic.oup.com/bioinformatics/article/34/8/1295/4708302#supplementary-data). 

From my understanding, the numerical labels for the fold categories in the data [correspond to these ones that you have uploaded here](https://github.com/multicom-toolbox/DeepSF/blob/master/datasets)/fold_label_relation2.txt) correct?, e.g. Fold label 36 would be a.37 in SCOP.  

Then, I would like to ask how do these labels (like a.37, b.12 etc.) link to [the actual Fold categories and their descriptions in the SCOP database here](https://scop.mrc-lmb.cam.ac.uk/) ?

Thank you!

**Comment by LA11131110128** [comment]
Yes, I got the same problem as well. Have you got the answer to this question?
